app/serializers/provider_serializer.rb

```diff
@@ -113,4 +105,17 @@ def select_value_for_provider(provider_code, values)
     index = provider_code.to_i(36) % values.count
     values[index]
   end
+
+  def generate_provider_contacts
+    object.contacts.map { |c| c.attributes.slice('type', 'name', 'email', 'telephone') }
+  end
+
+  def return_application_alert_recipient
```

Comment:
How about just simply `application_alert_recipient`? i.e. just a noun without attaching a verb/action to it, which is a bit more idiomatic Ruby. I don't think that clashes with anything, but if it does you could use `application_alert_recipient_record` or similar.

Reply:
application_alert_recipient worked fine 😄